iib/workers/tasks/utils.py

```diff
@@ -243,10 +243,8 @@ def reset_docker_config():
 def set_registry_token(token, container_image):
     """
     Configure authentication to the registry that ``container_image`` is from.
-
     This context manager will reset the authentication to the way it was after it exits. If
     ``token`` is falsy, this context manager will do nothing.
-
```

Comment:
Is there a particular reason why this empty line, and 2 above, were removed?

Reply:
nope were removed accidently